File: LatPrak2_2 (PY)/Product.py
class Product():
#kelas yang digunakan untuk mengimplementasikan kelas Product

	#atribut
	__price = "";
	__id_product = "";

	#constructor
	def __init__(self, price = "", id_product = ""):
		self.price = price
		self.id_product = id_product
		
		

	#get dan set untuk atribut price
	def setPrice(self, price):
	#mengeset nilai atribut price
		self.price = price

	def getPrice(self):
	#mengembalikan nilai atribut price
		return self.price
	
	#get dan set untuk atribut id product
	def setIdProduct(self, id_product):
	#mengeset nilai atribut id_Product
		self.id_product = id_product

	def getIdProduct(self):
	#mengembalikan nilai atribut id product
		return self.id_product
	
	

File: LatPrak2_1 (JAVA)/Product.java
class Product{
	//atribut
	private String price;
	private String id_product;

	//constructor
	Product(){
	}

	//get & set untuk atribut price 
	public void setPrice(String price){
		this.price = price;
	}
		
	public String getPrice(){
		return this.price;
	}

	//get & set untuk atribut id product
	public void setIdProduct(String id_product){
		this.id_product = id_product;
	}
		
	public String getIdProduct(){
		return this.id_product;
	}
}
